pkg/log/log_file_test.go

```diff
@@ -26,14 +26,13 @@ import (
 	"github.com/spf13/pflag"
 	"github.com/stretchr/testify/assert"
 
-	"k8s.io/component-base/logs"
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
 )
 
 const oneMB = 1 * 1024 * 1024
 
 var (
-	testFlags          = initFlags()
```

Comment:
was this creating an issue?

Reply:
This was creating issue I believe because it was being initialized before the init in logs, instead I just moved the init flags code into the tests themselves.

Reply:
makes sense